bosh-setup/manifests/multiple-vm-cf.yml

```diff
@@ -320,6 +456,153 @@ properties:
     port: 5524
     roles:
     - {name: ccadmin, password: c1oudc0w, tag: admin}
+  consul:
+    agent:
+      log_level: null
+      servers:
+        lan:
+        - REPLACE_WITH_CONSUL_IP
+    agent_cert: |+
+      -----BEGIN CERTIFICATE-----
+      MIIEITCCAgugAwIBAgIQTDMCJHBiQrY8hlM78doHTzALBgkqhkiG9w0BAQswEzER
```

Comment:
Actually in created VMs, we put these files under ~/example_manifests. For cloud foundry users, these files are for reference. If you think it is still not enough, we can add comments here.